airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/display_report.py

```diff
@@ -214,12 +213,7 @@
 
 METRICS_TYPE_TO_ID_MAP = {"campaigns": "campaignId", "adGroups": "adGroupId", "productAds": "adId", "targets": "targetId", "asins": "asin"}
 
-
-class Tactics(str, Enum):
-    T00001 = "T00001"
-    T00020 = "T00020"
-    T00030 = "T00030"
-    REMARKETING = "remarketing"
+TACTICS = ["T00020", "T00030"]
```

Comment:
Could you please leave some comments about reducing the elements of this list? Does the API require these 2 only?

Reply:
Yes, now only this 2 is available for sp report. Others deprecated since July 2022.